site/docs/guides/flowctl/create-derivation.md

```diff
@@ -24,88 +19,46 @@ If you don't have an account yet, [go to the web app](https://dashboard.estuary.
 * An existing Flow **collection**. Typically, you create this through a **capture** in the Flow web application.
 If you need help, see the [guide to create a Data Flow](../create-dataflow.md).
 
-* **flowctl** installed locally. For help, see the [installation instructions](../../concepts/flowctl.md#installation-and-setup).
-
-## Pull your specification files locally
+## Getting Started
 
-To begin working in your local environment, you must authenticate Flow from the command line.
-Then, you'll need to add your source collection's specification files to a **draft** and bring the draft into your local environment for editing.
+To create a derivation, navigate to the [Collections](https://dashboard.estuary.dev/collections) page in Flow, click on the **NEW TRANSFORMATION** button. This brings up a **Derive A New Collection** pop-up window.
 
-1. Authorize flowctl.
+Deriving a new collection consists of three steps:
 
-   1. Go to the [CLI-API tab of the web app](https://dashboard.estuary.dev/admin/api) and copy your access token.
+* **Step 1:** Select source collections: In the **Available Collections** dropdown, select the collection you want to derive.
 
-   2. Run `flowctl auth token --token <paste-token-here>`
+* **Step 2:** Transformation Language: There are two language options to select: **SQL** and **Typescript**.
 
-2. Begin by creating a fresh draft. This is where you'll add the specification files you need from the catalog.
+* **Step 3:** Write transformations: Give your derived collection a name. Then click the **PROCEED TO GITPOD** button.
 
-   ```console
-   flowctl draft create
-   ```
+This opens up GitPod in another tab, where an environment is already set up for you.
 
-   The output table shows the draft ID and creation time. It doesn't have any catalog entities in it yet.
-   You'll add the source collection's specification to the draft.
+## GitPod Set-Up
 
-3. Identify the collection (or collections) in the catalog that contains the data you want to derive, and add it to your draft.
-
-   ```console
-   flowctl catalog draft --name namespace/data-flow-name/my-collection
-   ```
+We integrate with GitPod in order to let users leverage the full capabilities of SQLite. GitPod is free to use. It is an online Integrated Development Environment (IDE) that provides a complete development environment that can be accessed through a web browser, with all the necessary tools and dependencies pre-installed.
```

Comment:
General stylistic tip: focus on talking directly to the reader ("you") and about what Flow can do (_not_ "we," the Estuary team). The goal is to focus them and their relationship with our platform, and to be digestible and not too stiff. 

To that end i'd suggest this change here: "Flow integrates with GitPod so you can leverage the full capabilities of SQLite."